packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -151,10 +154,26 @@ export const ActivityLibrary = (props: Props) => {
 
   return (
     <div className='flex h-full w-full flex-col bg-white'>
-      <ActivityLibraryHeader className='hidden md:flex' onClose={handleCloseClick}>
+      <ActivityLibraryHeader
+        className='hidden md:flex'
+        title='Start Activity'
+        leftNavigation={<LogoBlock className='flex-shrink-0 border-none' />}
+        rightNavigation={
+          <Link className='p-2' to={`/`} replace={true}>
+            <Close className='m-auto h-8 w-8' />
+          </Link>
+        }
+      >
```

Comment:
left/right navigation is a pattern borrowed from top app bar - which is basically the same component - https://medium.com/google-developer-experts/exploring-jetpack-compose-topappbar-c8b79893be34